app/models/manageiq/providers/vmware/infra_manager/metrics_capture.rb

```diff
@@ -369,33 +369,18 @@ def perf_query(params, counter_info, interval_name)
     return counter_values_by_mor_and_ts if params.blank?
 
     Benchmark.current_realtime[:num_vim_queries] = params.length
-    _log.debug { "Total item(s) to be requested: [#{params.length}], #{params.inspect}" }
 
-    query_size = concurrent_requests(interval_name)
```

Comment:
Interesting, this was the only thing `interval_name` was used for?  Can we drop it from the method params now?

Reply:
Yeah makes sense because `params` should already have the `interval` taken into account from up here https://github.com/ManageIQ/manageiq-providers-vmware/pull/871/files#diff-a8841609053fab02e1f864b18a14bc9dfbc0565ad1c4db98cf3dac8a217bd071R282